proposals/stac_update.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "stac_update",
```

Comment:
`stac_update` process has exactly the same summary and description as `export_collection` — do we need two processes at all or are they both doing the same thing?

Reply:
Sorry, my fault. I forgot to update the summary and description. Fixed.